app/app.go

```diff
@@ -150,12 +151,12 @@ func (a *App) TelemetryId() string {
 	return a.Srv().TelemetryId()
 }
 
-func (s *Server) HTMLTemplates() *template.Template {
+func (s *Server) TemplatesContainer() *templates.Container {
 	if s.htmlTemplateWatcher != nil {
-		return s.htmlTemplateWatcher.Templates()
+		return s.htmlTemplateWatcher
 	}
 
-	return nil
+	return templates.NewFromTemplate(template.New(""))
```

Comment:
Wondering if we still need this? We are failing fast now.

Reply:
I don't think so, the server has to have the container initialize a the beginning, so maybe we need to fix a couple of tests after this, bug let see what the CI thinks about that ;)

Reply:
Done